src/Model/Zone.php

```diff
@@ -52,9 +55,9 @@ class Zone extends DataObject
         "Enabled"
     ];
 
-	/**
-	 * {@inheritdoc}
-	 */
+    /**
+     * {@inheritdoc}
+     */
     public function populateDefaults()
     {
         parent::populateDefaults();
```

Comment:
The code below this is odd and potentially the cause of a bug

```
$current_region = Locale::getRegion(i18n::get_locale());
$this->Country = i18n::get_locale($current_region);
```

So `i8n::get_locale` does not take any input parameters which means `Country` is just going to be set to the current locale `en_GB` for instance. This means that the `getCountriesArray` will not work as expected which means the `onAfterWrite` will also not work as expected.